blocks/property-search-results/property-search-results.js

```diff
@@ -103,6 +103,7 @@ async function doSearch(search, redraw = true) {
     const controller = searchController;
     propertySearch(search).then((results) => {
       if (!controller.signal.aborted) {
+        window.propertyListings = results;
```

Comment:
Why are we storing this?

Reply:
So I don't have to go to the server to get basic information that we already have.